homeassistant/components/tuya/number.py

```diff
@@ -250,6 +250,21 @@
             icon="mdi:thermometer-lines",
         ),
     ),
+    # Humidifier
+    # https://developer.tuya.com/en/docs/iot/categoryjsq?id=Kaiuz1smr440b
+    "jsq": (
+        NumberEntityDescription(
+            key=DPCode.TEMP_SET,
+            name="Temperature",
+            icon="mdi:thermometer-lines",
+        ),
+        NumberEntityDescription(
+            key=DPCode.TEMP_SET_F,
+            name="Temperature Fahrenheit",
```

Comment:
Sometimes both are there, but in that case, the unit of measurement will help out
```suggestion
            name="Temperature",
```